contracts/BalanceTrackerFixedPriceBase.sol

```diff
@@ -106,6 +106,12 @@ abstract contract BalanceTrackerFixedPriceBase {
         // Get account balance
         uint256 balance = mapRequesterBalances[requester];
 
+        // Update balance with native value
```

Comment:
The native value was not handled correctly

Reply:
I don't like this, `Base` should not know about `msg.value` or not. I will refactor further.

Reply:
There's a `_checkNativeValue()` function above, at least need to move it there.